app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -170,20 +171,23 @@ class BulkUploadImport extends React.Component {
   }
 
   handleProjectSubmit(e) {
-    if(e && e.preventDefault) {
-      e.preventDefault();
-    }
-    this.clearError();
-    if(!this.isProjectInvalid()) {
-      this.addProject()
-    }
+    if (this.refs.new_project.inputRef.value.trim().length) {
+      $('.new-project-button').click();
```

Comment:
let's avoid using jquery in this way

Reply:
Yep, I agree we should avoid the refs, but refactoring all instance of ref in bulkupload and single upload components could be better done in a separate PR otherwise this PR would become large with unrelated code changes

Reply:
Ok for now. The "Input new project" dropdown is still not disappearing after I click "Create project" though? It should disappear after the project is created, like it used to.